qsub_loop.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
-for num_dir in {733..734}
+for dir_num in {882..882}
```

Comment:
I would pass these numbers as command line arguments, or even better, pass the directories as arguments using a glob.

Reply:
I realise you're using the `sigfig` variable when you name the job, swings and roundabouts, that would need to be changed.

Reply:
Yes, that would be nicer but I think it's good enough for this job and makes it easier to use `sigfig` in the job name.

Reply:
Still reckon this should be a command line argument. Do you plan invoking this as a post-processing script? In which case this flexibility is a must I would have thought.

Reply:
hm, good point, but I was thinking in terms of processing existing outputs. Processing as the run proceeds would need to be done after collation, which could get messy as collation doesn't always complete properly. The run is almost complete (8 years left) so I was planning to bit groom the backlog, finish the run, then bit-groom the new outputs.